pset-functions.js

```diff
@@ -208,6 +318,30 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+function doubleLetters (word) {
+  const wordLetters = word.split('')
+  let hasDoubleLetters = false
+
+  //  iterate through letters of the submitted word
+  wordLetters.forEach((letter, index) => {
+    //  declare the current letter and previous letter of the iteration
+    const currentLetter = wordLetters[index]
+    const previousLetter = wordLetters[index - 1]
+
+    //  debug
+    //  console.log(`current letter is ${currentLetter}. previous letter was ${previousLetter}.`)
+
+    //  if they're the same, then double letters are present,
+    //  return true
+    if (currentLetter === previousLetter) {
+      hasDoubleLetters = true
+    }
+  })
+
+  return (hasDoubleLetters)
+}
+
+console.log(doubleLetters('apple'))
```

Comment:
@sad3xoxo Great job, the solution here looping through the elements and comparing current and previous to determine if the use case has a repeating letter.
take a look at an alternative approach [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)